web/src/scripts/webflow/fundingSync.mjs

```diff
@@ -252,14 +253,20 @@ const deleteFundings = async () => {
       return await deleteItem(funding.id, fundingCollectionId);
     } catch (error) {
       console.error(error.response.data);
+      if (error.response.status === 429){
+        console.debug("Catching rate limit error hit")
+        await wait(65000);
```

Comment:
Why 65000? Buffer?

Reply:
Buffer. But yes, we should probably call this out as a global const.